metricbeat/module/windows/services/services_windows.go

```diff
@@ -0,0 +1,370 @@
+// +build windows
+
+package services
+
+import (
+	"bytes"
+	"strconv"
+	"syscall"
+	"unicode/utf16"
+	"unsafe"
+
+	"github.com/elastic/beats/libbeat/common"
+	gosigar "github.com/elastic/gosigar/sys/windows"
+	"github.com/pkg/errors"
+	"golang.org/x/sys/windows"
+
+	"github.com/elastic/beats/winlogbeat/sys"
+)
+
+// Windows API calls
+//sys _OpenSCManager(machineName *uint16, databaseName *uint16, desiredAcces ServiceSCMAccessRight) (handle ServiceDatabaseHandle, err error) = advapi32.OpenSCManagerW
+//sys _EnumServicesStatusEx(handle ServiceDatabaseHandle, infoLevel ServiceInfoLevel, serviceType ServiceType, serviceState ServiceEnumState, services *byte, bufSize uint32, bytesNeeded *uint32, servicesReturned *uint32, resumeHandle *uintptr, groupName *uintptr) (err error) [failretval==0] = advapi32.EnumServicesStatusExW
+//sys _OpenService(handle ServiceDatabaseHandle, serviceName *uint16, desiredAccess ServiceAccessRight) (serviceHandle ServiceHandle, err error) = advapi32.OpenServiceW
+//sys _QueryServiceConfig(serviceHandle ServiceHandle, serviceConfig *byte, bufSize uint32, bytesNeeded *uint32) (err error) [failretval==0] = advapi32.QueryServiceConfigW
+//sys _CloseServiceHandle(handle uintptr) (err error) = advapi32.CloseServiceHandle
+
+var (
+	sizeOfEnumServiceStatusProcess = (int)(unsafe.Sizeof(EnumServiceStatusProcess{}))
```

Comment:
Writing `sizeof` instead of `sizeOf` is most common (b/c that's the name of the C operator that generates the value).